build.sbt

```diff
@@ -83,8 +83,10 @@ lazy val commonLib = project("common-lib").settings(
     // needed to parse conditional statements in `logback.xml`
     // i.e. to only log to disk in DEV
     // see: https://logback.qos.ch/setup.html#janino
-    "org.codehaus.janino" % "janino" % "3.0.6"
-  ),
+    "org.codehaus.janino" % "janino" % "3.0.6",
+    "org.mockito" % "mockito-core" % "2.18.0"
```

Comment:
Shouldn't this be:
```suggestion
    "org.mockito" % "mockito-core" % "2.18.0" % Test
```

Reply:
This doesn't work for me, can I get a minute tomorrow to look at this please?